source/cloud/nvidia/brev.md

```diff
@@ -4,41 +4,66 @@ review_priority: "p0"
 
-You can get up and running with RAPIDS in a few quicks thanks to the Brev RAPIDS quickstart.
+## Brev Instance Setup 
+There are two ways that you can get up and running with RAPIDS in a few quicks thanks to the Brev RAPIDS quickstart:
+1. Brev GPU Instances - quickly get the GPU, across most clouds, to get your work done.
+2. Brev Launchables - quickly create one-click starting, reusable instances that you customized to your MLOps needs.
 
-## Setting up your instance
+### 1. Setting up your Brev GPU Instance
 
-Navigate to the [Brev console](https://console.brev.dev/) and click on "Create your first instance".
+1. Navigate to the [Brev console](https://console.brev.dev/) and click on "Create your first instance".
 
 ![Screenshot of the "Create your first instance" UI](/_static/images/platforms/brev/brev1.png)
 
-Select "Container Mode".
+2. Select "Container Mode".
 
 ![Screenshot showing "Container Mode" highlighted](/_static/images/platforms/brev/brev2.png)
 
-Attach the "NVIDIA RAPIDS" Container.
+3. Attach the "NVIDIA RAPIDS" Container.
 
 ![Screenshot showing the "NVIDIA RAPIDS" container highlighted](/_static/images/platforms/brev/brev3.png)
 
-Configure your own instance.
+4. Configure your own instance.
 
 ![Screenshot of the "Create your Instance" UI with an NVIDIA T4 GPU highlighted](/_static/images/platforms/brev/brev4.png)
 
-And hit "Deploy".
+5. And hit "Deploy".
 
 ![Screenshot of the instance creation summary screen with the deploy button highlighted](/_static/images/platforms/brev/brev5.png)
 
+### 2. Setting up your Brev Launchable
+1. Click Create Launchable
+1. Click **Compute** and select your GPU based on GPU type, number of GPUs, the cloud provider, and/or budget you have.  It is good to understand your GPU requirements before picking an instance, or use it to figure out which instance.  Once selected, click **Save** 
+1. Click ** Container**.  When adding the **Container**, you can use the NVIDIA RAPIDS Container, use Docker Compose and edit our example yaml so that it preinstalls any additional conda or pip packages into your container before entry.  
+  1. If you just need standard NVIDIA RAPIDS install, Select **Container Mode > NVIDIA RAPIDS Container**. If using the Base Container, you may need to preinstall Jupyter.  If using the Notebooks Container, **Do not** Preinstall Jupyter, as that will break your instance.
+  1. If you need a customized environment, with additional packages on top of NVIDIA RAPIDSm use **Docker Compose** When using Docker Compose, you can upload a docker-compse yaml file.  Here is an example docker-compose file, **[docker/brev/docker-compose-nb-2412.yaml](https://github.com/clara-parabricks-workflows/single-cell-analysis-blueprint/raw/main/docker/brev/docker-compose-nb-2412.yaml)** that you can use as your base.  **Do not** Preinstall Jupyter when using that file, as it already will be installed.
```

Comment:
I think linking to the example is fine here. If we want to create a more bespoke example that walks you through doing a specific thing on brev then we should create another page in the workflow examples section.